mne/preprocessing/tests/test_scalp_coupling_index.py

```diff
@@ -0,0 +1,64 @@
+# Authors: Robert Luke <mail@robertluke.net>
+#          Eric Larson <larson.eric.d@gmail.com>
+#          Alexandre Gramfort <alexandre.gramfort@inria.fr>
+#
+# License: BSD (3-clause)
+
+import os.path as op
+
+import pytest
+import numpy as np
+
+from mne.datasets.testing import data_path
+from mne.io import read_raw_nirx
+from mne.preprocessing.nirs import optical_density, scalp_coupling_index
+from mne.datasets import testing
+
+fname_nirx_15_0 = op.join(data_path(download=False),
+                          'NIRx', 'nirx_15_0_recording')
+fname_nirx_15_2 = op.join(data_path(download=False),
+                          'NIRx', 'nirx_15_2_recording')
+fname_nirx_15_2_short = op.join(data_path(download=False),
+                                'NIRx', 'nirx_15_2_recording_w_short')
+
+
+@testing.requires_testing_data
+@pytest.mark.parametrize('fname', ([fname_nirx_15_2_short, fname_nirx_15_2,
+                                    fname_nirx_15_0]))
+@pytest.mark.parametrize('fmt', ('nirx', 'fif'))
+def test_scalp_coupling_index(fname, fmt, tmpdir):
+    """Test converting NIRX files."""
+    assert fmt in ('nirx', 'fif')
+    raw = read_raw_nirx(fname)
+    raw = optical_density(raw)
+    sci = scalp_coupling_index(raw)
+
+    # All values should be between -1 and +1
+    assert (sci < 1.0).all()
```

Comment:
I would use https://docs.scipy.org/doc/numpy-1.10.0/reference/generated/numpy.testing.assert_array_less.html

It will give a nicer message on error IIRC

Reply:
Thanks. Done